src/charts/utilization/utilization-chart.html

```diff
@@ -1,7 +1,7 @@
 <div class="utilization-chart-pf">
-    <span class="title">{{config.title}}</span>
+    <span class="h3">{{config.title}}</span>
```

Comment:
I made the suggested change and I have added ngDoc only overrides for the .contents h# specifications to make them consistent with PF to give an accurate look in the ngDocs. I added one spec override for the font-weight for the utilization card h3 title as it varies from PF h3.